src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/services.po.ts

```diff
@@ -28,15 +28,24 @@ export class ServicesPageHelper extends PageHelper {
   addService(serviceType: string, exist?: boolean) {
     cy.get(`${this.pages.create.id}`).within(() => {
       this.selectServiceType(serviceType);
-      if (serviceType === 'rgw') {
-        cy.get('#service_id').type('rgw.foo');
-        cy.get('#count').type('1');
-      } else if (serviceType === 'ingress') {
-        this.selectOption('backend_service', 'rgw.rgw.foo');
-        cy.get('#service_id').should('have.value', 'rgw.rgw.foo');
-        cy.get('#virtual_ip').type('192.168.20.1/24');
-        cy.get('#frontend_port').type('8081');
-        cy.get('#monitor_port').type('8082');
+      switch (serviceType) {
+        case 'rgw':
+          cy.get('#service_id').type('foo');
+          cy.get('#count').type('1');
+          break;
+
+        case 'ingress':
+          this.selectOption('backend_service', 'rgw.rgw.foo');
+          cy.get('#service_id').should('have.value', 'rgw.rgw.foo');
+          cy.get('#virtual_ip').type('192.168.20.1/24');
+          cy.get('#frontend_port').type('8081');
+          cy.get('#monitor_port').type('8082');
+          break;
+
+        case 'nfs':
+          cy.get('#service_id').type('testnfs');
+          cy.get('#count').type('1');
+          break;
```

Comment:
`nfs` and `rgw` has the same step for creating the service right? why not make them one and just pass the service name from the test suite?

Reply:
I'd rather prefer to keep it separate for different service_types as any further changes required here can be easily done.